src/NUnitFramework/framework/Attributes/TestCaseAttribute.cs

```diff
@@ -71,8 +62,9 @@ public TestCaseAttribute(params object[] arguments)
         /// <param name="arg"></param>
         public TestCaseAttribute(object arg)
         {
-            this.RunState = RunState.Runnable;			
-            this.Arguments = new object[] { arg };
+            RunState = RunState.Runnable;			
+            Arguments = new object[] { arg };
+            Properties = new PropertyBag();
```

Comment:
Yeah, I intended to go back but got involved with some other stuff and ended up just pushing it.

One of my mantras for teams I work with is that you have to create duplication before you can remove it. It has some real benefit to actually see the duplicate code first rather than just doing the "removal" in your head so that it never gets created. OTOH, if you forget to go back... :-(